pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "openai < 2",
     "oracledb < 3",
     "philter-lite < 1",
-    "pyarrow < 19",
+    "pyarrow < 18",
```

Comment:
I'm not 100% sure that dependabot will do something sane with the way our constraints are set up with maximums. Hopefully it will! But as a test, I'm also dropping pyarrow down a notch for the moment. Let's see what it does.